site/content/contribute/more-info/getting-started/contribution-checklist.md

```diff
@@ -16,22 +16,24 @@ Follow this checklist for submitting a pull request (PR):
 
 1. You've signed the {{< newtabref href="https://mattermost.com/mattermost-contributor-agreement/" title="Contributor License Agreement" >}}, so you can be added to the Mattermost {{< newtabref href="https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true" title="Approved Contributor List" >}}.
     - If you've included your mailing address in the signed {{< newtabref href="https://mattermost.com/mattermost-contributor-agreement/" title="Contributor License Agreement" >}}, you may receive a {{< newtabref href="https://forum.mattermost.com/t/limited-edition-mattermost-mugs/143" title="Limited Edition Mattermost Mug" >}} as a thank you gift after your first pull request is merged.
-2. Your ticket is a Help Wanted GitHub issue for the Mattermost project you're contributing to.
+2. You have claimed the ticket that you wish to work on by asking for an assignment from the Mattermost team.
+   - Tickets are assigned on a first-come-first-serve basis.
+4. Your ticket is a Help Wanted GitHub issue for the Mattermost project you're contributing to.
```

Comment:
@Rutam21 - I think the step numbering is off by one from this point to the end.

Reply:
Blocking on this